src/lib/viewers/image/ImageViewer.js

```diff
@@ -78,6 +80,23 @@ class ImageViewer extends ImageBaseViewer {
         super.handleAssetAndRepLoad();
     }
 
+    /**
+     * Sets mode to be AnnotationMode.NONE if the zoomed image overflows the viewport.
+     *
+     * @return {void}
+     */
+    handleZoomEvent(height, width) {
+        const viewport = {
+            width: this.wrapperEl.clientWidth - 2 * IMAGE_PADDING,
+            height: this.wrapperEl.clientHeight - 2 * IMAGE_PADDING,
+        };
+
+        // If the image is overflowing the viewport, we should set annotation mode to be NONE so that the user can pan
+        if (width > viewport.width || height > viewport.height) {
+            this.processAnnotationModeChange(this.annotationControlsFSM.transition(AnnotationInput.CANCEL));
```

Comment:
Should this be `AnnotationInput.RESET`? I only see a transition for `AnnotationInput.CANCEL` from `AnnotationState.REGION_TEMP`

Reply:
Yep it should be RESET instead.